sdk/deviceupdate/azure-iot-deviceupdate/src/samples/java/com/azure/iot/deviceupdate/GetDeviceSample.java

```diff
@@ -5,28 +5,31 @@
 import com.azure.core.exception.HttpResponseException;
 import com.azure.core.http.policy.HttpLogDetailLevel;
 import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
 import com.azure.core.util.Configuration;
 import com.azure.identity.DefaultAzureCredentialBuilder;
 
-public class GetUpdateNotFoundSample {
+public class GetDeviceSample {
     public static void main(String[] args) {
-        DeviceUpdateAsyncClient client = new DeviceUpdateClientBuilder()
+        DeviceManagementAsyncClient client = new DeviceManagementClientBuilder()
             .endpoint(Configuration.getGlobalConfiguration().get("AZURE_ACCOUNT_ENDPOINT"))
             .instanceId(Configuration.getGlobalConfiguration().get("AZURE_INSTANCE_ID"))
             .credential(new DefaultAzureCredentialBuilder().build())
             .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
             .buildAsyncClient();
 
-        // BEGIN: com.azure.iot.deviceupdate.DeviceUpdateAsyncClient.notfound
         try {
-            client.getUpdateWithResponse("foo", "bar", "0.0.0.1", null).block();
+            Response<BinaryData> response = client.getDeviceWithResponse(
+                Configuration.getGlobalConfiguration().get("DEVICEUPDATE_DEVICE"),
+                null).block();
```

Comment:
The common usage on sync API is to create `DeviceManagementClient` via `.buildClient()`.

And then API call does not need to have `.block()` or `.toIterable()` to convert async response to sync response.

Reply:
Got it. Sorry I am not Java person, I am just mostly doing copy/paste. :-(